libs/blocks/global-footer/global-footer.js

```diff
@@ -69,16 +69,38 @@ class Footer {
 
     // TODO: log to LANA if Footer content could not be found
     if (!this.body) return;
-    // TODO: revisit region picker and social links decoration logic
+
     const regionAnchor = this.body.querySelector('.region-selector a');
-    if (regionAnchor?.href) {
-      regionAnchor.setAttribute('href', `${regionAnchor.getAttribute('href')}#_dnt#_dnb`);
+    const regionNextSibling = regionAnchor?.nextSibling;
+    const regionParent = regionAnchor?.parentNode;
+    const detachedRegion = regionParent?.removeChild(regionAnchor);
+
+    const socialLinks = Array.from(document.querySelectorAll('.social a'));
```

Comment:
```suggestion
    const socialLinks = [...document.querySelectorAll('.social a')];
```

Reply:
`socialLinks` returns as an empty array for me

Reply:
Fixed